Doc/whatsnew/3.14.rst

```diff
@@ -312,6 +312,9 @@ ctypes
 * The :exc:`~ctypes.COMError` exception is now public.
   (Contributed by Jun Komoda in :gh:`126686`.)
 
+* The :func:`~ctypes.CopyComPointer` function is now public.
```

Comment:
You might want to say "On Windows, [...]" (you can also amend the entry for `COMError` just for consistency).

Reply:
Good catch!

Explicitly stating that COM-related features are specific to Windows helps prevent misunderstandings and ensures that Linux and Mac users don't waste time searching for solutions when these imports fail.

Reply:
That's exactly what I had in mind. Because, knowing myself, I would have clicked on the link and be sad that it's Windows only (well.. not really sad but I would be caught off guard!)

Reply:
To reduce such unfortunate situations, the [`comtypes` README includes an FAQ about non-Windows platforms](https://github.com/enthought/comtypes/blob/2dc69bd663b3e0f735afda55270380792fbd76a9/README.md?plain=1#L55-L66).
 
The same should be clearly stated for `ctypes` as well.